java-squid/src/main/java/org/sonar/java/resolve/TypeAndReferenceSolver.java

```diff
@@ -131,8 +132,10 @@ private static void completeMetadata(JavaSymbol symbol, List<AnnotationTree> ann
     for (AnnotationTree tree : annotations) {
       AnnotationInstanceResolve annotationInstance = new AnnotationInstanceResolve((JavaSymbol.TypeJavaSymbol) tree.symbolType().symbol());
       symbol.metadata().addAnnotation(annotationInstance);
-      if (tree.arguments().size() > 1 || (!tree.arguments().isEmpty() && tree.arguments().get(0).is(Tree.Kind.ASSIGNMENT))) {
-        for (ExpressionTree expressionTree : tree.arguments()) {
+      Arguments arguments = tree.arguments();
+      if (arguments != null) {
+        if (arguments.size() > 1 || (!arguments.isEmpty() && arguments.get(0).is(Tree.Kind.ASSIGNMENT))) {
+        for (ExpressionTree expressionTree : arguments) {
           AssignmentExpressionTree aet = (AssignmentExpressionTree) expressionTree;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this line start at column 13. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AIndentationCheck)